pkg/client/cache/delta_fifo.go

```diff
@@ -199,6 +258,7 @@ func (f *DeltaFIFO) queueActionLocked(actionType DeltaType, obj interface{}) err
 		return KeyError{obj, err}
 	}
 	newDeltas := append(f.items[id], Delta{actionType, obj})
```

Comment:
It might have had a value, that's why check here https://github.com/kubernetes/kubernetes/pull/14881/files#diff-b32f14c6f3cce65061c1673064448839R268 and stomp here https://github.com/kubernetes/kubernetes/pull/14881/files#diff-b32f14c6f3cce65061c1673064448839R273 .

If there was a slice in the map that had enough capacity, the `append` would modify the existing map and `newDeltas` and `f.items[id]` would both point it.

If there wasn't a slice in the map or there wasn't enough capacity, the `append` would try to `realloc` and would `newDeltas` and `f.items[id]` would point to different locations and have different content.